### PR TITLE
PHP 8.0 support 🚀 and Laravel 6, 7 and 8 support

### PR DESCRIPTION
**Pull request by tortuetorche**
* Add PHP 7.3, 7.4 and 8.0 support
* Add Laravel 6, 7 and 8 support
* Update phpspec tests
* Drop PHP 5.6, 7.0 and 7.1 support
* Drop Laravel 5 support

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,13 +3,14 @@ language: php
 sudo: false
 
 php:
-  - 5.6
-  - 7.0
-  - 7.1
   - 7.2
+  - 7.3
+  - 7.4
+  - 8.0snapshot
```
Comment (PatrickRose) [inline .travis.yml:7]: Surprised that Travis doesn't have a proper PHP8 thing already!

**Review by PatrickRose — COMMENTED** [review]
All looks good - though does #320 still apply?

**Comment by PatrickRose** [comment]
In fact - there's a lot of tickets about incompatabilities with laravelcollective/html - how many of them still apply?

File: .travis.yml (hunk 1 above)
Comment (tortuetorche) [reply .travis.yml:7]: Last time I checked, they don't! But soon I hope 👍

**Comment by tortuetorche** [comment]
> All looks good - though does #320 still apply?

Good catch, I think we can remove the `getToken()` support and just keep `token()`

Reference: https://laravel.com/docs/5.4/upgrade

**Comment by tortuetorche** [comment]
FYI build (from my fork) passed: https://travis-ci.com/github/tortuetorche/bootstrapper/builds/205628104

By the way, you should [migrate](https://docs.travis-ci.com/user/migrate/legacy-services-to-github-apps-migration-guide/) from the legacy https://travis-ci.org to the new https://travis-ci.com, to get faster builds 👍
I did it for my repositories, it's like night and day 😆

**Comment by tortuetorche** [comment]
> In fact - there's a lot of tickets about incompatabilities with laravelcollective/html - how many of them still apply?

I need to do more manual tests to answer you

**Comment by tortuetorche** [comment]
> All looks good - though does #320 still apply?

Done!

Build (from my fork) passed: https://travis-ci.com/github/tortuetorche/bootstrapper/builds/205632538

File: .travis.yml (hunk 1 above)
Comment (tortuetorche) [reply .travis.yml:7]: We have to wait the [`php-build`](https://github.com/php-build/php-build) 1.0.0 release I think...
References:
* https://travis-ci.community/t/php-8-0-missing/10132
* https://github.com/php-build/php-build/pull/651

**Comment by PatrickRose** [comment]
> By the way, you should migrate from the legacy https://travis-ci.org to the new https://travis-ci.com, to get faster builds 👍

I'd love to, but it looks like because this is still under @patricktalmadge's name it's something they'd have to do. With the amount of work that gets done on Bootstrapper on a daily basis (I no longer use Laravel on a day to day basis), it's probably not worth it :(

**Comment by tortuetorche** [comment]
> > In fact - there's a lot of tickets about incompatabilities with laravelcollective/html - how many of them still apply?
> 
> I need to do more manual tests to answer you

Good news, I tested the `Form::select()`, `Form::submit()`, `Alert`, `Button`  and `Icon` components without any problems!

**Comment by tortuetorche** [comment]
Build passed: https://travis-ci.org/github/patricktalmadge/bootstrapper/builds/746964699

**Comment by PatrickRose** [comment]
Awesome, I'll merge this later tonight and tag it for a release.

Thanks so much for your contribution!

**Comment by tortuetorche** [comment]
@PatrickRose You've have done much much more than me on this package, so thanks to you!

> Awesome, I'll merge this later tonight and tag it for a release.
> 
> Thanks so much for your contribution!

You've have done much much more than me on this package, so big thank to you!

**Comment by PatrickRose** [comment]
And tagged - thanks again!

**Comment by tortuetorche** [comment]
> And tagged - thanks again!

FYI, the [5.12.0](https://github.com/patricktalmadge/bootstrapper/releases/tag/5.12.0) tag isn't present in Packagist (https://packagist.org/packages/patricktalmadge/bootstrapper)
Maybe you need to force update on the Packagist side?
As a workaround, I use this in my `composer.json` file:

```javascript
    "require": {
        //...
        "patricktalmadge/bootstrapper": "dev-master",
        //...
    },
```

**Comment by tortuetorche** [comment]
@PatrickRose I think you can do a manual update on https://packagist.org/packages/patricktalmadge/bootstrapper

If you login with your GitHub account, then you can click on the `Update` green button, just under the package's title.
I did it for a package who I don't have administrator permission 👍


Here a screenshot:
packagist

**Comment by PatrickRose** [comment]
I've done the update, sorry about that!

**Pull request merged**